cps-wallet-connector/README.md

```diff
@@ -0,0 +1,221 @@
+Connection standards and APIs should be explicitly versioned to clearly allow upgrades.
+Furthermore, ideally APIs should allow for optional extendability to facilitate specialization.
+Clear scope of versions / extensions.
+- Should consider tiers of wallet - can be tricky to see adoption for dApps, dApss prefer lowest denominator.
+
+#### 6. Respect the role of wallet
+APIs should be written to have a clear scope, understanding the potential strains placed on wallet providers.
+APIs, at a base level, should focus on preserving the base [role of the wallet](#clear-role-of-wallet).
+
+### Context
+Here we will add context to the current state of Cardano's wallet connectors. 
+
+#### Cardano Wallet-Web Bridge History
+[CIP-30](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0030/README.md) is *the* fundamental wallet connector standard for Cardano.
+This standard has facilitated the emergence of dApp development on Cardano by defining both a connection standard and an API.
+It is, to date, the only wallet connector standard to see wide adoption in the ecosystem.
+Using an injected Javascript object for communication between wallets and web-based stacks.
+Since its' authoring CIP-30 has seen continued iteration, with changes to its API.
+
+##### CIP-30 Alternatives
+Since CIP-30's authoring there have only been two other competing standards, in CIP-45 | Decentralized WebRTC dApp-Wallet Communication ([#395](https://github.com/cardano-foundation/CIPs/pull/395)) and CIP-90 | Extendable dApp-Wallet Web Bridge ([#462](https://github.com/cardano-foundation/CIPs/pull/462/)).
+Although neither of of this standards have seen wide adoption.
+
+Although not a direct competitor, CIP-13 could be seen as an alternative standard.
+This standard
+
+##### CIP-30 Extensions
+- CIP-62
+- CIP-95
+- CIP-103
+- CIP-104 -- impact of sharing account public key?
+
+#### Historical Issues
+- Lack of ecosystem cohesion around Role of the Wallet
+- Combination of connection and API in one standard
+- Lack of ecosystem focus, wallet expected to do a lot -> not clearly defined what the role is
+- Encouraged emulated wallets
+- Typescript APIs limiting
+- No way for dApp to subscribe to wallet events
+
+#### Solutions
+- CIP-30 - extendibility mechanism
+- CIP-30 - not changing anymore
+
+## Use cases
+<!-- A concrete set of examples written from a user's perspective, describing what and why they are trying to do. When they exist, this section should give a sense of the current alternatives and highlight why they are not suitable. -->
+
+### NFT Marketplace
+Alice wishes to buy a NFT from an smart contract based NFT marketplace for an agreed price because she wishes to support the artist.
+Alice wants to be able to use a familiar website interface where she can browse rendered NFTs then select and buy on the same site.
+
+Without the dApp and her wallet communicating it is very difficult to construct the needed transaction to buy the NFT Alice wants.
+
```

Comment:
End users are not users of wallet connectors. They are users of wallets. We should instead describe what dApp developers and wallet implementers expect from connectors standard.

It's pointless to describe connectors from end users point of view, because we can't enforce that the wallet does not steal funds for example.

Reply:
This is an interesting perspective, I think youre right.
I need to add more examples from such perspectives although perhaps it is nice to keep a couple examples from the users perspective?